drivers/r2dbc-driver/src/main/kotlin/app/cash/sqldelight/driver/r2dbc/R2dbcDriver.kt

```diff
@@ -155,7 +156,7 @@ fun CoroutineScope.R2dbcDriver(
 }
 
 // R2DBC uses boxed Java classes instead primitives: https://r2dbc.io/spec/1.0.0.RELEASE/spec/html/#datatypes
-class R2dbcPreparedStatement(private val statement: Statement) : SqlPreparedStatement {
```

Comment:
Exposed to allow casting to a driver specific Statement to access driver methods.